docker-utils/startup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -xe
+
+cfgTpl=/usr/local/bin/custom-cfg.sh
+cfgHome=/usr/local/zeebe/conf/zeebe.cfg.toml
+cfgLog=/usr/local/zeebe/conf/log4j2.xml
+
+if [[ "$DEPLOY_ON_KUBERNETES" == "true" ]]; then
+
+    DNSNAME="${HOST}.${DNS}"
```

Comment:
As these variables are required but maybe not set we could also use the `-u` bash option to fail startup if one of the variables is not defined, i.e. `#!/bin/bash -xeu`. We would then have to define default values for `DEPLOY_ON_KUBERNETES` and `DEBUG` in the Dockerfile, i.e. `false`, which might be a good idea to make these variables more discoverable .

Reply:
:+1: